src/test/unit-tests/GrappaRegistry.t.sol

```diff
@@ -65,6 +67,27 @@ contract GrappaRegistry is Test {
         assertEq(collateral, address(weth));
         assertEq(collatDecimals, 18);
     }
+
+    function testReturnOptionDetailsFromTokenId() public {
+        uint256 _expiry = block.timestamp + 14 days;
+        uint256 _strikePrice = 4000 * UNIT;
+
+        grappa.registerAsset(address(weth));
+
+        uint40 _product = grappa.getProductId(address(0), address(0), address(weth), address(0), address(weth));
```

Comment:
I'm currently only prefixing function arguments with `_`. Maybe better to keep things consistent.